### PR TITLE
Creating an account now sends an activation e-mail

### PR DESCRIPTION
**Pull request by parkersprouse**

### CODE REVIEW

File: apps/web/controllers/auth/activate_account/show.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+          handle_exception AuthFormError => :handle_error
+
+          before :must_not_be_authenticated
+          before do
+            raise AuthFormError, 'Invalid activation token' if !params.valid? || user.blank?
+          end
+          before do
+            raise AuthFormError, 'This account has already been activated' if user.pw_hash.present?
+          end
```
Comment (abrayard) [inline apps/web/controllers/auth/activate_account/show.rb:21]: Just wanted to call out that I really like this error handling. V clean

File: config/environment.rb [hunk 2]
```diff
@@ -26,21 +26,19 @@
     logger level: :debug
 
     mailer do
-      # When starting to use SparkPost for mail handling, uncomment the below
-      # delivery :smtp, address: ENV.fetch('SMTP_HOST'), port: ENV.fetch('SMTP_PORT'),
-      #   user_name: 'SMTP_Injection', password: ENV.fetch('SPARKPOST_SECRET'),
-      #   authentication: :login
+      delivery :smtp, address: ENV.fetch('SMTP_HOST'), port: ENV.fetch('SMTP_PORT'),
+                      user_name: 'SMTP_Injection', password: ENV.fetch('SPARKPOST_SECRET'),
+                      authentication: :login
```
Comment (abrayard) [inline config/environment.rb:31]: Logically I understand why this is spaced the way it is, but I personally don't like it visually. I don't care either way; we just need to make sure we all adhere to a standard one way or the other from here on out.

Comment (parkersprouse) [reply config/environment.rb:31]: I'm not necessarily a huge fan of this style either, but this is what the linter currently wants. We'd have to alter / disable the rule if we decide we don't want to go with this style

Comment (abrayard) [reply config/environment.rb:31]: https://docs.rubocop.org/rubocop/cops_layout.html#enforcedstyle-with_fixed_indentation
Seems like this is what we want

**Review by abrayard — APPROVED** [review]
Looks good, approving whether you want to do the rubocop rule update or not

**Pull request merged**